deployments/xpumanager_sidecar/base/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- https://github.com/intel/xpumanager/deployment/kubernetes/daemonset/base/?ref=V1.2.29
```

Comment:
not related to this PR but this version seems rather old?

Reply:
I'll update it to the latest.